src/libultra/gu/rotate.c

```diff
@@ -1,5 +1,4 @@
-#include <ultra64.h>
-#include <global.h>
+#include "global.h"
 
 void guRotateF(float m[4][4], float a, float x, float y, float z) {
     static float D_80097F90 = 0.017453292; // TODO this is PI / 180.0, we should define pi somewhere
```

Comment:
```suggestion
    static float D_80097F90 = M_PI / 180;
```

Reply:
Is there a reason `float` is used instead of `f32`? There's also a small number of functions in `functions.h` using `float` instead of `f32`

Reply:
My guess is it's a libultra thing, but I don't know for certain.

Reply:
libultra should use the typedefs too, a lot of things need cleaning up but I think the PR is already large enough and this is quite out of scope for this one.